packages/@cdktf/hcl2cdk/test/helpers/convert.ts

```diff
@@ -15,9 +15,10 @@ import {
   ConstructsMakerProviderTarget,
   readSchema,
 } from "@cdktf/provider-generator";
-
 import deepmerge from "deepmerge";
 
+const includeSynthTests = Boolean(process.env.CI);
```

Comment:
My personal opinion would be to invert how this works and to make tests that run synth opt-out instead. We could e.g. add this opposite flag to the NPM `watch` script to make it easier to use. To me the default behavior should be the one that runs more tests and is as closely resembling what happens in CI as possible. Doing different things locally than in CI by default is a recipe for struggle (especially for contributors trying to run tests locally).

Reply:
I have no strong opinion here, but I struggled with folks not using my carefully crafted tools and flags if they were not the default for local development, that's why I biased in the favour of the fast solution. 

Summoning @mutahhir as the tie breaker.

Reply:
<img src="https://media1.giphy.com/media/8PySWh7GnrgNxdKZEv/giphy.gif"/>

### Ahem

I hereby declare the tie broken! 

I would also prefer the local development thing be default, and using CI as a special case. Primarily because I'll forget to set the flag and will also not remember what that flag was named, whereas CI has a pretty good memory.

Reply:
I'm fine with going that path if the two of you prefer a different default :)